alerter.c

```diff
@@ -1,6 +1,8 @@
 #include <stdio.h>
 #include <assert.h>
 
+typedef int (*networkAlert_fnptr)(float);
+
 int alertFailureCount = 0;
 
 int networkAlertStub(float celcius) {

```

Comment:
both test and production alert code is part of the same file. It needs to be in logically separate files.